rollup-cli/src/actions/offchain/send.js

```diff
@@ -34,14 +68,20 @@ async function send(urlOperator, idTo, amount, walletJson, password, tokenId, us
         onChain: 0,
         newAccount: 0,
     };
-
     walletRollup.signRollupTx(tx); // sign included in transaction
     const parseTx = stringifyBigInts(tx);// convert bigint to Strings
-
     const resTx = await apiOperator.sendTx(parseTx);
+    let nonceObjectToWrite;
+    if (resTx.status.toString() === '200' && nonce === undefined) {
+        nonceObjectToWrite = _addNonce(nonceObject, currentBatch, nonceToSend);
+    } else if (resTx.status.toString() === '200' && nonce !== undefined) {
+        nonceObjectToWrite = [{ batch: currentBatch, nonce: nonce + 1 }];
```

Comment:
maybe just save the currenbatch and the last nonce is enough? I dont know if it's encessary to store a vector of nonces of the last batches